armory/scenarios/multimodal_so2sat_scenario.py

```diff
@@ -60,35 +60,42 @@ def _evaluate(
             estimator = load_defense_internal(config["defense"], estimator)
 
         attack_config = config["attack"]
-        attack_channels = attack_config.get("generate_kwargs", {}).get("channels")
+        attack_channels = attack_config.get("generate_kwargs", {}).get("mask")
 
         if attack_channels is None:
             if self.attack_modality == "sar":
                 logger.info("No mask configured. Attacking all SAR channels")
-                attack_channels = range(4)
+                attack_channels = np.concatenate(
+                    (np.ones(4, dtype=np.float32), np.zeros(10, dtype=np.float32)),
+                    axis=0,
+                )
             elif self.attack_modality == "eo":
                 logger.info("No mask configured. Attacking all EO channels")
-                attack_channels = range(4, 14)
+                attack_channels = np.concatenate(
+                    (np.zeros(4, dtype=np.float32), np.ones(10, dtype=np.float32)),
+                    axis=0,
+                )
             elif self.attack_modality == "both":
                 logger.info("No mask configured. Attacking all SAR and EO channels")
-                attack_channels = range(14)
+                attack_channels = np.ones(14, dtype=np.float32)
 
         else:
             assert isinstance(
                 attack_channels, list
```

Comment:
I think it would make sense here to also assert that the list is (1) of length 14 and (2) only contains 0/1. This would catch when a user incorrectly tries to specify channels e.g. [0,1,2,3] rather than a mask [1, 1, 1, 1, 0, ..., 0]